README.adoc

```diff
@@ -454,6 +455,48 @@ tomcat:
 <4> Optional parameter to configure sleeping time between log downloading. You can set in seconds using _s_ or miliseconds using _ms_. By default time unit is miliseconds and value 500.
 <5> Optional parameter to configure number of retries to be done. By default 10 iterations are done.
 
+Custom Await strategy:
+
+[source, java]
+.org.arquillian.cube.docker.impl.await.CustomAwaitStrategyImpl
+----
+public class CustomAwaitStrategyImpl implements AwaitStrategy {
+
+  Await params;
+  DockerClientExecutor dockerClientExecutor;
+  Cube<?> cube;
+
+  public void setCube(Cube<?> cube) {
```

Comment:
@aslakknutsen I am not creating the await with all parameters because not all await strategy requires all of them, if you look at current implementations you will see that some of them requires one, two or three parameters, so instead of creating an interface with a method with three parameters, I think it is better to maintain it clean and let the users choose what they want to inject.